packages/global/components/blocks/content-load-more.marko

```diff
@@ -2,11 +2,13 @@ $ const {
   id,
   type,
   name,
+  primarySectionAlias,
```

Comment:
@Shinsina  This should be something like displayAds.  The conditional logic related to primary section should live in calling file and tell this file to display ads.